### PR TITLE
Fix TS errors in linkTo.tsx

### PR DESCRIPTION
**Pull request by GHaberis**
## What are you doing in this PR?

Typescript error squashing in `linkTo.tsx`.

### CODE REVIEW

File: support-frontend/assets/pages/paper-subscription-landing/components/content/linkTo.tsx [hunk 1]
```diff
@@ -34,14 +37,16 @@ function LinkTo({
...
 	isPricesTabLink?: boolean;
-}) {
+}): JSX.Element {
+	console.log('tab --->', tab);
+
 	return (
 		<Link
 			css={isPricesTabLink ? linkStyles(tab, activeTab) : linkColor}
-			href={paperSubsUrl(tab === 'delivery')}
+			href={paperSubsUrl(tab === 'HomeDelivery')}
```
Comment (GHaberis) [inline support-frontend/assets/pages/paper-subscription-landing/components/content/linkTo.tsx:49]: Interestingly I thought I'd picked up a bug here. The code was incorrect as `tab` is the type `PaperFulfilmentOptions` which can only ever hold the value "HomeDelivery" or "Collection". I fixed the code and the went looking for the bug, but I found it never manifests itself to the user...

`linkTo` is used for the bottom tabs on the Newspaper landing page, the bottom tabs are anchor tags but we don't use the `href` property of these anchors to route a user to a new page when they click on them, instead we have an `onClick` handler that take control, the actions of the handler are correct - the correct Home Delivery or Collection content is displayed in the page. This differs from the way the top tabs on the Newspaper landing page behave, these are also anchor tags but they route a user through to either `/subscribe/paper` or `subscribe/paper/delivery`.

**Top tabs**
Screenshot 2022-04-01 at 14 48 50


**Bottom tabs**
Screenshot 2022-04-01 at 14 49 04

Comment (i-hardy) [reply support-frontend/assets/pages/paper-subscription-landing/components/content/linkTo.tsx:49]: Weird 🤔 I feel like it's not great if the two separate sets of tabs have slightly different behaviour, but as long as the user sees the correct thing I suppose it's not a big deal.

**Review by i-hardy — APPROVED** [review]
Looks good, just a stray log statement to clear up

File: support-frontend/assets/pages/paper-subscription-landing/components/content/linkTo.tsx (hunk 1 above)
Comment (GHaberis) [reply support-frontend/assets/pages/paper-subscription-landing/components/content/linkTo.tsx:49]: I know, I don't think we should have 2 sets of behaviours either. I'm going to create a card to look into consolidating one way or the other.

**Pull request merged**